### DISCUSSION TITLE
I can't access the dashboard when using reverse proxy from nginx

### DISCUSSION THREAD

**Discussion by MADS0LADEN** [plan-player-analytics/Plan#2404] [Archive]
I've been trying to get this working for a while now!
I can't get the reverse proxy from nginx working with the dashboard...

I don't know what else to write or show because i've read the wiki a dozen times and tried a lot of different things for the last couple of months...

Nginx config:
<img width="1437" alt="Code_2021-03-01_20-39-54" src="https://user-images.githubusercontent.com/4103363/109549454-4b4b2680-7ace-11eb-971b-c2741925e36d.png">

Network settings on the machine:
<img width="960" alt="chrome_2021-03-01_20-47-07" src="https://user-images.githubusercontent.com/4103363/109550286-5e122b00-7acf-11eb-9e30-eb0ced20b618.png">

Plan config:
<img width="605" alt="Code_2021-03-01_20-49-59" src="https://user-images.githubusercontent.com/4103363/109550560-bfd29500-7acf-11eb-8a39-3745a9111a07.png">

DNS records:
<img width="433" alt="chrome_2021-03-01_21-01-47" src="https://user-images.githubusercontent.com/4103363/109551862-61a6b180-7ad1-11eb-8056-435ecb7fb767.png">
I've tried with cloudflare proxy on and off.
And tried with A record of "analytics" with CF proxy on/off

**Comment by AuroraLS3** [comment]
You have so much in front of the panel that it is difficult to debug where the issue is, it could be any of these (This is the stack the connection needs to go through on your setup):
```
Cloudflare DNS (Does it point to the right place?)
     Cloudflare Proxy server (Does it point to the right place?)
         Firewall (Is nginx accessible?)
         Nginx reverse-proxy (Does it point to the right place?)
             Pterodactyl docker container (Is the port routed correctly?)
                 Plan webserver (Is the webserver running?)
```

I'd say start at the bottom and once you get the first step working then start piling on top more proxies.

Easiest first check is to see if you can access the webserver on the machine the server is running on:
```
curl localhost:8804
```
- If it fails, your docker routing has issues
- If it succeeds, one less issue to worry about, focus your efforts on the nginx

**Comment by AuroraLS3** [comment]
Assuming you got this sorted out
